index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>garden-vue</title>
+    <link rel="stylesheet" href="/static/normalize.css">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/static/semantic.min.css">
+    <script src="/static/semantic.min.js"></script>
```

Comment:
Given the build system style of `vue` projects with webpack, please avoid including resources in this way.

Take a look at [these docs](https://simulatedgreg.gitbooks.io/electron-vue/content/docs/using_css_frameworks.html) for a solid example of using a framework.